lightspeed/flame
# Move storybook down to package level. Update to 5.3

## Description

Slowly cleaning things up globally. Moving the storybook down will slim down our top level package and let us understand just what is actually needed vs what isn't.

Each package should have their own storybook instance, or else we end up with this super fat top level package.json and an even more complex webpack config for storybook, not to mention we increase complexity in what goes where in the monorepo.

I've also taken time to cleanup some things (flush useless test, shift files around to be less exposed globally.).

Also took time to port over some stories (namely the one in `flame-tokens`) into `flame`. Important to note that those stories are kinda coded ugly, but we'll be moving to MDX anyways in a few, so this is fine.





## How to test?

- Checkout branch, run `yarn && yarn bootstrap && yarn dev`
- Open [Storybook](http://localhost:6006)
- Or check the deploy preview on Netlify (link available in comments)

## Checklist
